apps/desktop/src/app/sync/dynamic-connector.ts

```diff
@@ -59,6 +56,11 @@ export class DynamicConnectorWrapper implements ISourceConnector {
     }
   }
 
+  getParametersValues(): ConnectorParameters {
+    //TODO
```

Comment:
is this something you forgot or which is planned for later?

Reply:
dynamic connectors are off for now, I need to put some thinking into it, they will be implemented in a future PR